src/OrchardCore/OrchardCore.OpenId.Core/OpenIdExtensions.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using OpenIddict.Abstractions;
+using OrchardCore.Data.Migration;
+using OrchardCore.OpenId.Abstractions.Managers;
+using OrchardCore.OpenId.Services.Managers;
+using OrchardCore.OpenId.YesSql.Indexes;
+using OrchardCore.OpenId.YesSql.Migrations;
+using OrchardCore.OpenId.YesSql.Models;
+using OrchardCore.OpenId.YesSql.Resolvers;
+using OrchardCore.OpenId.YesSql.Stores;
+using YesSql.Indexes;
+
+namespace Microsoft.Extensions.DependencyInjection
```

Comment:
Doing that is not mandatory, but it's a cool and convenient way to allow OpenIddict users to use YesSql stores in their non-OC apps.